tempodb/encoding/config.go

```diff
@@ -11,6 +11,7 @@ type BlockConfig struct {
 	IndexDownsampleBytes int              `yaml:"index_downsample_bytes"`
 	IndexPageSizeBytes   int              `yaml:"index_page_size_bytes"`
 	BloomFP              float64          `yaml:"bloom_filter_false_positive"`
+	BloomFilterShardSize int              `yaml:"bloom_filter_shard_size"`
```

Comment:
See other comment, should probably be `bloom_filter_shard_size_bytes` to follow convention.